l10n_nl_company_type/data/l10n_nl_company_types.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="company_type_stichting" model="res.partner.company.type">
+            <field name="name">Stichting</field>
```

Comment:
this is a translatable field, so add the data in English and add a translation file